lib/spack/spack/build_systems/cuda.py

```diff
@@ -82,10 +82,10 @@ def cuda_flags(arch_list):
     # https://docs.nvidia.com/cuda/archive/9.0/cuda-installation-guide-linux/index.html
     # https://docs.nvidia.com/cuda/archive/8.0/cuda-installation-guide-linux/index.html
 
-    arch_platform = ' arch=ppc64le platform=linux'
+    arch_platform = ' target=ppc64le: platform=linux'
     # information prior to CUDA 9 difficult to find
     conflicts('%gcc@6:', when='+cuda ^cuda@:9' + arch_platform)
-    conflicts('%gcc@8:', when='+cuda ^cuda@10.0.130' + arch_platform)
+    conflicts('%gcc@8:', when='+cuda ^cuda@:10.2' + arch_platform)
```

Comment:
This is not up to date, CUDA 10.1+ supports GCC 8 :)

Reply:
I suggest
```
     conflicts('%gcc@8:', when='+cuda ^cuda@:10.0' + arch_platform) 
     conflicts('%gcc@9:', when='+cuda ^cuda@:10.2' + arch_platform) 
```

Reply:
Would that be correct for cuda-10.0.130 which only supports up to gcc-7?
https://github.com/spack/spack/pull/13472#issuecomment-547093042

Reply:
Yes, the case of CUDA 10.0 is covered by the first line, which says "if the spec is CUDA 10.0 or earlier, then all GCC are fine besides GCC 8 or newer."

Reply:
I have probably got myself confused, but would 10.0.130 be considered later than 10.0 and thus not caught by that? Or, is 10.0.130 still considered 10.0?

Reply:
@glennpj It will be considered later. The algorithm is quite complex and I lost track of the corner cases we deal with in there, but the part that takes care of making `10.0` be less than `10.0.130` is this one:

https://github.com/spack/spack/blob/0a5c184a5dc76f0a85650e9f42cce94251d9791a/lib/spack/spack/version.py#L308-L310

Reply:
Oh, in that case it should read
```
     conflicts('%gcc@8:', when='+cuda ^cuda@:10.0.999' + arch_platform) 
     conflicts('%gcc@9:', when='+cuda ^cuda@:10.2.999' + arch_platform) 
```